content/applications/hr/referrals/share_job.rst

```diff
@@ -0,0 +1,163 @@
+*Referrals* application.
+
+.. image:: share-jobs/share.png
+   :align: center
+   :alt: The various sharing icons that appear for each job.
+
+Link
+----
+
+To share the job position with a customized tracking link, click the :guilabel:`Share Now` button
+with the :icon:`fa-chain` :guilabel:`(link)` icon inside it. A :guilabel:`Link to Share` pop-up
+window appears with the tracking link. Click :guilabel:`Copy` to copy the link. After the link is
+copied, click the :guilabel:`Close` button to close the pop-up window. Next, share the link with
+the prospective employee.
+
+Facebook
+--------
+
+To share the job position using Facebook, click the :guilabel:`Share Now` button with the
+:guilabel:`f (Facebook)` icon inside it.
+
+If the user is already logged into Facebook, when the :guilabel:`f (Facebook)` icon is clicked, a
+:guilabel:`Share on Facebook` page loads in a new tab, with the link populated in the main body of
+the new post in a pop-up window. If the user is *not* already logged in, a log-in screen loads,
+instead, prompting the user to log-in to Facebook first.
+
+Type in any additional information to add to the post, then share the job position using the
+available options in Facebook.
+
+X (formerly Twitter)
+--------------------
+
+A job position can also be shared on X (formerly Twitter). Click the :guilabel:`Share Now` button
+with the :guilabel:`X` icon inside it.
+
+If the user is already signed in to X (formerly Twitter), when the :guilabel:`Share Now` icon is
+clicked, an X (formerly Twitter) page loads in a new tab with a pre-populated message ready to post,
+in a draft pop-up window. If the user is *not* already signed in, a sign-in screen loads instead,
+prompting the user to sign in to X (formerly Twitter).
+
+The default message is:
+
+:guilabel:`Amazing job offer for (Job Position)! Check it live: (link to Job Position)`
+
+Type in any additional information, or make any edits to the message, then share using the available
+options in X (formerly Twitter).
+
+LinkedIn
+--------
+
+To share a job position on LinkedIn, click the :guilabel:`Share Now` button with the :guilabel:`in
+(LinkedIn logo)` inside it.
+
+If the user is already logged into LinkedIn, when the :guilabel:`Share Now` button is clicked, a new
+tab loads in LinkedIn, with a link to the job position at the top. If the user is *not* already
+logged in, a log-in screen loads instead, prompting the user to log-in to LinkedIn first.
+
+The job position can be shared either in a public post, or in a private message to an individual (or
+group of individuals).
+
+Type in any additional information, or make any edits to the message or post, then share using the
+available options in LinkedIn.
+
+.. _referrals/email-jobs:
+
+Email a friend
+--------------
+
+Another way to share job opportunities is to share the entire current list of open job positions
+instead of one job position at a time. To do this, navigate to the main *Referrals* app dashboard.
```

Comment:
```suggestion
instead of one job position at a time. To do this, navigate to the :menuselection:`Referrals` main dashboard.
```
I think since you are telling them to navigate somewhere, this gets a menuselection tag